sbncode/CAFMaker/CAFMaker_module.cc

```diff
@@ -996,6 +999,22 @@ void CAFMaker::produce(art::Event& evt) noexcept {
   // Fill detector & reco
   //#######################################################
 
+  //Beam gate and Trigger info
+  fSRTrigger.clear();
```

Comment:
Why does this need to be a member variable rather than just creating an SRTrigger object within the produce() function here?

Reply:
Or indeed just fill straight into `rec.hdr.triggerinfo`?